src/mobu/services/business/nublado.py

```diff
@@ -166,6 +185,27 @@ async def startup(self) -> None:
                 self.logger.warning(msg)
 
     async def execute(self) -> None:
+        try:
+            await self._execute()
+        except Exception as exc:
+            monkey = None
+            event = "execute_code"  # Fallback
+            if isinstance(exc, MobuMixin):
+                if exc.monkey:
+                    monkey = exc.monkey
+                if exc.event:
+                    event = exc.event
```

Comment:
What about something like:
```suggestion
            monkey = getattr(exc, "monkey", None)
            event = getattr(exc, "event", "execute_code")
```